### ISSUE TITLE
Failed validity check on `papirus-folders`

### ISSUE THREAD

**Issue by sgoudham** [catppuccin/aur-packages#37]
Greetings
Im still getting a failed validity check when trying to install papirus-folders-catppuccin-git using an AUR helper ([paru](https://github.com/Morganamilo/paru))

The error seems to be validating papirus-folders checksum fails
```
==> Validating source files with md5sums...
    papirus-folders-catppuccin-git ... Skipped
    papirus-folders.hook ... Passed
    papirus-folders ... FAILED
```

Checking the [PKGBUILD](https://aur.archlinux.org/packages/papirus-folders-catppuccin-git) in the AUR for papirus-folders-catppuccin-git, i can see that the checksum for papirus-folder is not SKIP, but instead `07398d48f6d86df3573fafc612c9c198`. I dont know if this repo links to a different AUR package or not.

But in any case, the md5 sum for papirus-folders is not wrong. I downloaded papirus-folders from the link from the link in PKGBUILD and `md5sum` the downloaded file. The result is here:

```
md5sum papirus-folders
07398d48f6d86df3573fafc612c9c198  papirus-folders
```

Which matches with the PKGBUILD of papirus-folders-catppuccin-git.

I dont know why it is failing.

Host details:
Arch x86_64
6.9.4-arch1-1
paru v2.0.3 - libalpm v14.0.0

_Originally posted by @sudoAlphaX in https://github.com/catppuccin/aur-packages/issues/34#issuecomment-2167268322_

**Comment by sgoudham** [comment]
I think the latest PR should have fixed this issue @sudoAlphaX.

Feel free to comment if it's not and I can re-open this issue!

**Comment by sudoAlphaX** [comment]
papirus-folders-catppuccin-git now installs without any errors using paru AUR helper, and I can see that papirus-folders checksum is being skipped.

Thank you for your help @sgoudham.
Have a great rest of your day!

**Comment by sudoAlphaX** [comment]
This is so silly.
I am not getting any output from executing `papirus-folders` command.

I uninstalled papirus-folders-catppuccin-git and installed papirus-folders-git, and compared the /usr/bin/papirus-folders file, which is what i got from executing `whereis papirus-folders`.

The papirus-folders executable differs for the two packages.

Here are the contents of the executable from papirus-folders-catppuccin-git: https://pastebin.com/bRxAuEje
And here are the contents from papirus-folders-git: https://pastebin.com/WynnL17K

The contents are very much different from each other:  https://pastebin.com/LHJutx80

Sorry to bother you once again

**Comment by sudoAlphaX** [comment]
I do not have much experience with the Arch build system, but ill try my best to find the issue and open a PR if i can.

**Comment by ghostx31** [comment]
I figured out the issue. 

Turns out, since the CI only pushed `PKGBUILD` and `.SRCINFO`, the old `papirus-folders` script was still present in the AUR repo upstream. 

Can you try building this again and let me know if this fails now or not?

**Comment by sudoAlphaX** [comment]
Nope, still facing the same issue. I just uninstalled papirus-folders-git along with configuration files and protected data using -Rcns flag, and cleared pacman and paru cache using -Sc. I then reinstalled catppuccin-folders-catppuccin-git. Nothing changed.

**Comment by sudoAlphaX** [comment]
I do not understand why this line exists in the PKGBUILD:
`  "papirus-folders::https://raw.githubusercontent.com/PapirusDevelopmentTeam/papirus-folders/master/papirus-folders")` and i download that file and its the proper papirus-folders script, but if the same is done using paru, it fetches another papirus-folders script.

Maybe its my AUR helper that has bugs. Let me install yay and report the results.

**Comment by ghostx31** [comment]
We need the file since upstream papirus script now supports custom colours and we add it as a source, and apparently it works correctly in our and ci and even tested successfully by someone else on our discord server.

I'm pretty confused about why this is happening now.

**Comment by sudoAlphaX** [comment]
This seems to be an issue specific to me. I just created an arch vm (fresh install) and installed paru in it. Then I installed papirus-folders-catppuccin-git. `papirus-folders` command is working well.

**Comment by sudoAlphaX** [comment]
I just installed [yay](https://github.com/Jguer/yay) (Another AUR Helper) on my main machine and installed papirus-folders-catppuccin-git using yay, and now, `papirus-folders` command is again working correctly.

**Comment by sudoAlphaX** [comment]
I uninstalled papirus-folders-catppuccin-git (installed using yay) and cleared cache of paru and tried installing again. Same result. Somehow, that particular app is not getting installed using paru on my specific machine, because it worked in the VM.
Really interesting...

**Comment by sudoAlphaX** [comment]
I tried reinstalling paru but still got the same result.
But when I tried installing papirus-folders-catppuccin-git with verbose flag, i noticed this line:
`Targets   : /home/alpha/.cache/paru/clone/papirus-folders-catppuccin-git/papirus-folders-catppuccin-git-r30.f83671d1-2-any.pkg.tar.zst`

I thought maybe the cache wasnt getting flushed, so i just deleted everything in $HOME/.cache folder and tried reinstalling the tool again.
And this time, it finally worked. 'papirus-folders' command was giving a proper output, and im able to change the folder colors.

What had happened is paru kept trying to use the pre-downloaded .tar.gz to reinstall the utility.

Anyways, thank you very much for your support @ghostx31. Have a wonderful weekend.
